nomad/csi_endpoint.go

```diff
@@ -142,7 +142,7 @@ func (v *CSIVolume) List(args *structs.CSIVolumeListRequest, reply *structs.CSIV
 				}
 
 				// Filter on the request namespace to avoid ACL checks by volume
```

Comment:
Do we still need this conditional if we're now querying the state store by namespace?